### PR TITLE
hw-2 done

### PR DESCRIPTION
**Pull request by TrashboxeZ**

### CODE REVIEW

File: awscli/change_ip.sh
Comment (saaverdo) [inline awscli/change_ip.sh:1]: Nice solution

File: awscli/curl_resp.txt
Comment (saaverdo) [inline awscli/curl_resp.txt:1]: `curl -I ...` will return just header which is enough for this task.
(just reduce the wall os text :) ) 
Anyway, it's OK 👍

File: awscli/init.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+CREATE USER IF NOT EXISTS 'admin'@'%' IDENTIFIED BY 'Pa55WD';
+SELECT user FROM mysql.user;
+create database flask_db;
+grant ALL on flask_db.* to 'admin'@'%';
+SHOW DATABASES;
+
+
+
+export MYSQL_USER="admin"
```
Comment (saaverdo) [inline awscli/init.sql:9]: These env vars are required on APP instance, not DB one

**Review by saaverdo — APPROVED** [review]
Nice work!
Approved 👍

File: awscli/curl_resp.txt
Comment (TrashboxeZ) [reply awscli/curl_resp.txt:1]: typical 1 in shell curl -I

File: awscli/init.sql (hunk 1 above)
Comment (TrashboxeZ) [reply awscli/init.sql:9]: copied here temporary, bucket doesn't have those lines

**Pull request merged**